README.md

```diff
@@ -16,6 +16,13 @@ To view or contribute Charts, head over to the [charts repo](https://github.com/
 
 ![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
 
+**If you are a Helm 0.1 user** you will need to do an extra step when
+you upgrade to Helm 0.2. We changed our GitHub org from `deis` to
+`helm`, which means the new default charts repository is now
+`github.com/helm/charts`. To fix quickly, simply run `helm repo rm
+charts` followed by `helm repo add charts
+https://github.com/helm/charts`.
+
```

Comment:
Maybe create a code block? Something like: To fix quickly, simply run:

```
helm repo rm charts
helm repo add charts https://github.com/helm/charts
```

that way people can just copy/paste both commands in one fell swoop